POEApi.Model/Gear.cs

```diff
@@ -86,7 +86,21 @@ public override string PobData
 
                     if (Sockets != null && Sockets.Any())
                     {
-                        var socketData = string.Join("=", Sockets.Select(x => x.ToPobFormat()));
+                        //Links are denoted by "Groups"
```

Comment:
Add periods to the end of the comments (since you started them with capital letters).